routesrv/eskipbytes.go

```diff
@@ -61,11 +65,23 @@ func (e *eskipBytes) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 	span := tracing.CreateSpan("serve_routes", r.Context(), e.tracer)
 	defer span.Finish()
 
+	if r.Method != "GET" && r.Method != "HEAD" {
+		w.WriteHeader(http.StatusMethodNotAllowed)
+		return
+	}
+
 	data, etag, lastModified, initialized := e.bytes()
 	if initialized {
 		w.Header().Add("Etag", etag)
 		w.Header().Add("Content-Type", "text/plain; charset=utf-8")
-		http.ServeContent(w, r, "", lastModified, bytes.NewReader(data))
+		w.Header().Add(routing.RoutesCountName, strconv.Itoa(e.count))
+
+		switch r.Method {
+		case http.MethodHead:
+			w.WriteHeader(http.StatusOK)
+		case http.MethodGet:
+			http.ServeContent(w, r, "", lastModified, bytes.NewReader(data))
```

Comment:
http.ServeContent supports HEAD method so I think we'd only need to add X-Count header